.github/workflows/MERGE_MASTER_TO_DEVELOP.yml

```diff
@@ -0,0 +1,32 @@
+name: MERGE_MASTER_TO_DEVELOP
+on:
+  push:
+    branches:
+    - "master"
+
+jobs:
+  Merge_master_to_develop:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+
+    steps:
+    - name: Checkout develop
+      uses: actions/checkout@v2
+      with:
+        ref: develop
+        fetch-depth: 0
+    - name: Merge master to develop and push
+      run: |
+        git config user.name github-actions
+        git config user.email github-actions@github.com
+        git merge -m 'Merge master to develop' --no-edit origin/master
+        git push
+    - name: Notify on Slack if merge fails
+      uses: slackapi/slack-github-action@v1.15.0
+      if: failure()
+      with:
+        channel-id: ${{ secrets.SLACK_CHANNEL_ID }}
+        slack-message: "Could not merge <https://github.com/${{ github.repository }}/tree/${{ github.ref }}|${{ github.ref }}> to <https://github.com/${{ github.repository }}/tree/develop|${{ github.repository }}#develop>."
```

Comment:
Proposal: Rename to something more actionable:

```
Automatic merge of <https://github.com/${{ github.repository }}/tree/${{ github.ref }}|${{ github.ref }}> to <https://github.com/${{ github.repository }}/tree/develop|${{ github.repository }}#develop failed.
```

Reply:
```suggestion
        slack-message: "Automatic merge of <https://github.com/${{ github.repository }}/tree/${{ github.ref }}|${{ github.ref }}> to <https://github.com/${{ github.repository }}/tree/develop|${{ github.repository }}#develop failed."
```